TribalWars/src/ApplicationCore/Entities/BuildingRequiredBuilding.cs

```diff
@@ -10,8 +10,8 @@ public class BuildingRequiredBuilding
     {
         public int Id { get; set; }
         public int BuildingId { get; set; }
-        public int RequiredBuildingId { get; set; }
-        public int Level { get; set; }
         public virtual Building Building { get; set; }
+        public int RequiredBuildingId { get; set; }
+        public virtual RequiredBuilding RequiredBuilding { get; set; }
```

Comment:
It will be more readable if you will separate entity values from other entities inclusions, e.q.:
`public int Id { get; set; }`
`public int BuildingId { get; set; }`
`public int RequiredBuildingId { get; set; }`

`public virtual RequiredBuilding RequiredBuilding { get; set; }`
`public virtual Building Building { get; set; }`

Reply:
As Szymon explained to me, he said that such a division is clearer because it is known what it is used for.